packages/synapse-react-client/src/types/tanstack-table-augmentation.d.ts

```diff
@@ -23,9 +23,9 @@ declare module '@tanstack/react-table' {
 
     /** The type of filter that this column may have */
     filterVariant?: 'enumeration' // could be extended to include 'range', 'freeText', etc.
-    /** If the filterVariant is `enumeration`, this list captures the potential values that may be picked for filtering */
-    enumValues?: ColumnFilterEnumValue<TValue>[]
```

Comment:
Removed in favor of using one of the native `getFacetedUniqueValues` API. With this API, we can either 
* supply our own implementation that computes facet statistics (or retrieves them from a server)
* use the default implementation provided by TanStack Table that computes facet statistics client-side